Makefile

```diff
@@ -309,6 +319,7 @@ build-pytorch-cpu: build-cpu-py-310-base
 		$(CPU_PYTORCH_TAGS) \
 		--push \
 		.
+	docker run --rm -v `pwd`/tests:/workspace/tests -it $(DOCKERHUB_REGISTRY)/$(CPU_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH) /bin/bash -c "pip install pytest && pytest -m pytorch /workspace/tests"
```

Comment:
These multi-arch images seem to be pulling with an error `Requested platforms linux/amd64 do not match result platforms linux/amd64,linux/arm64`
You might have to add the `--platform` arg to the run command for this, and `tensorflow-cpu`.

Reply:
Is this similar to how we use `--platform` when build? I don't see the error though